src/validation.cairo

```diff
@@ -229,7 +229,7 @@ fn validate_coinbase(block: @Block, total_fees: u64, block_height: u32) -> Resul
     assert(*tx.inputs[0].previous_output.vout == 0xFFFFFFFF, 'vout should be 0xFFFFFFFF');
 
     // Ensure the input's TXID is zero
-    assert(*tx.inputs[0].previous_output.txid == 0, 'txid should be 0');
+    assert(*tx.inputs[0].previous_output.txid == Hash { value: [0; 8] }, 'txid should be 0');
```

Comment:
here is actually a case where we could have this `new` function that generates a default Hash.

@maciejka  what do you prefer? leaving it this is way or adding a new method for that?

Reply:
Implement Default?

Reply:
ok!